packages/scss/src/components/status/component.scss

```diff
@@ -0,0 +1,47 @@
+@mixin component {
+	display: inline-flex;
+	align-items: center;
+	white-space: nowrap;
+
+	@at-root {
+		.status-dot {
+			aspect-ratio: 1;
+			width: var(--spacings-smaller);
+			border-radius: 50%;
+			background-color: currentColor;
```

Comment:
with @RipripKipkip we use box-shadow instead of background-color, like : box-shadow: 0 0 0 var(--palettes-error-600);

Reply:
This is another possible approach indeed. I liked the idea here of being able to pass a text colour with currentColor.